src/components/Footer.jsx

```diff
@@ -70,13 +70,31 @@ class SiteFooter extends React.Component {
                 </li>
               </ul>
             </div>
+
+            <nav className="nav-colophon" aria-label="About">
+              <ol>
+                <li><a href={`${config.LMS_BASE_URL}/about`}>About Us</a></li>
+                <li><a href={`${config.LMS_BASE_URL}/blog`}>Blog</a></li>
+                <li><a href={`${config.LMS_BASE_URL}/donate`}>Donate</a></li>
+                <li><a href={`${config.LMS_BASE_URL}/tos`}>Terms of Service</a></li>
+                <li><a href={`${config.LMS_BASE_URL}/privacy`}>Privacy Policy</a></li>
+                <li><a href={`${config.LMS_BASE_URL}/help`}>Help</a></li>
+                <li><a href={`${config.LMS_BASE_URL}/contact`}>Contact Us</a></li>
+              </ol>
+            </nav>
```

Comment:
This will cause some conflicting behavior across legacy and MFE pages. In tutor-indigo, footer links on legacy pages can be toggled (https://github.com/overhangio/tutor-indigo/blob/release/tutorindigo/templates/indigo/lms/templates/footer.html). In here, the links will show all the times. The behavior needs to be consistent.